src/vue/backend/src/templates/AgentsSettings.vue

```diff
@@ -0,0 +1,129 @@
+                id="website-wrapper"
+                :value="directorySlug"
+                @change="agentSettingsStateChange({ key: directorySlug, value: $event.target.value })"
+            />
+        </idx-form-group>
+        <idx-form-group>
+            <idx-block className="control-label"><strong>Custom Wrapper</strong> Detailed sentence or two describing how custom wrappers can be used and how to set them up properly.</idx-block>
+        </idx-form-group>
+        <idx-form-group>
+            <idx-form-label customClass="control-label" for="website-wrapper">Wrapper Start HTML</idx-form-label>
+            <idx-form-input
+                type="text"
+                id="website-wrapper"
+                :value="wrapperStart"
+                @change="agentSettingsStateChange({ key: wrapperStart, value: $event.target.value })"
+            />
+        </idx-form-group>
+        <idx-form-group>
+            <idx-form-label customClass="control-label" for="website-wrapper">Wrapper End HTML</idx-form-label>
+            <idx-form-input
+                type="text"
+                id="website-wrapper"
+                :value="wrapperEnd"
+                @change="agentSettingsStateChange({ key: wrapperEnd, value: $event.target.value })"
+            />
+        </idx-form-group>
+    </idx-block>
+</template>
+<script>
+import { mapState, mapActions } from 'vuex'
+export default {
+    name: 'agentSettings',
+    data () {
+        return {
+            cssLabel: 'Deregister IMPress Agents Main CSS?'
+        }
+    },
+    computed: {
+        ...mapState({
+            deregisterMainCss: state => state.agentSettings.deregisterMainCss,
+            numberOfPosts: state => state.agentSettings.numberOfPosts,
+            directorySlug: state => state.agentSettings.directorySlug,
+            wrapperStart: state => state.agentSettings.wrapperStart,
+            wrapperEnd: state => state.agentSettings.wrapperEnd
+        })
+    },
+    methods: {
+        ...mapActions({
+            agentSettingsStateChange: 'agentSettings/agentSettingsStateChange'
+        })
+    }
+}
+</script>
+<style lang="scss">
+@import '~bootstrap/scss/forms';
+@import '~@idxbrokerllc/idxstrap/dist/styles/components/customSelect.scss';
+@import '~@idxbrokerllc/idxstrap/dist/styles/components/toggleSlider.scss';
+.fieldset {
+    // Global Styles
+    --space-1: 4px;
+    --space-2: 8px;
+    --space-3: 12px;
+    --space-4: 16px;
+    --space-5: 20px;
+    --space-6: 24px;
+    --font-size-label: 16px;
+    --font-size-p: 16px;
+    --line-height-label: 22px;
+    --line-height-p: 28px;
+    font-size: var(--font-size-p);
+    line-height: var(--line-height-p);
+    input[type=color], input[type=date], input[type=datetime-local], input[type=datetime], input[type=email], input[type=month], input[type=number], input[type=password], input[type=search], input[type=tel], input[type=text], input[type=time], input[type=url], input[type=week], select, textarea {
```

Comment:
Why do you have so many specifications on this? We are not using an input other than text

Reply:
this is to override some styles at the Wordpress level